src/effects/VST3/internal/ConnectionProxy.h

```diff
@@ -0,0 +1,42 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file ConnectionProxy.h
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+#include <thread>
+#include <pluginterfaces/vst/ivstmessage.h>
+
+namespace internal
+{
+   //!Host's proxy object between connection points
+   //Though it's not neccessary to place proxy, it's recommended to do so.
+   //Right now the only "useful" task performed by this proxy is ensuring
+   //that messages are sent on the same thread on which proxy object itself
+   //was created
```

Comment:
Doxygen extracts the brief description but not the detailed description.  Keep this separation of brief and detailed, and rewrite the detailes inside

```
/*!
...
*/
```